pkg/kubelet/lifecycle/handlers.go

```diff
@@ -55,6 +56,9 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
+		if err != nil {
+			glog.V(1).Infof("Liveness-Probe for Container %q in Pod %q failed - %q", buffer.String())
```

Comment:
This is not liveness probe, but lifecycle hooks. Change the subject in the commit as well  :)